src/main/java/org/folio/fqm/migration/strategies/V13_5CustomFieldRename.java

```diff
@@ -26,17 +26,17 @@
 import static org.jooq.impl.DSL.field;
 
 /**
- * Version 13 -> 14, handles custom field renaming.
+ * Version 13.5 -> 14, handles custom field renaming.
  *
  * The custom field naming scheme was changed in MODFQMMGR-376. This migration handles updating custom field names to match the new scheme.
  *
  * @see https://folio-org.atlassian.net/browse/MODFQMMGR-642 for the addition of this migration
  */
 @Log4j2
 @RequiredArgsConstructor
-public class V13CustomFieldRename implements MigrationStrategy {
+public class V13_5CustomFieldRename implements MigrationStrategy {
 
-  public static final String SOURCE_VERSION = "13";
+  public static final String SOURCE_VERSION = "13.5";
```

Comment:
V12 still produces a V13 with these changes, leading to V12 -> V13 -> 💥 with no path forward.

I'd suggest changing line 57 `applies` implementation to `"13".equals(version) || "13.5".equals(version)` or similar. The `SOURCE_VERSION` here does not do anything besides get used in that method